### PR TITLE
Build_deploy & integration job fixed

### PR DESCRIPTION
**Pull request by vibhutikumar07**
Build_deploy & integration job fixed.
cf toll was not getting install, which is fixed with this changes.

### CODE REVIEW

File: .github/workflows/deploy_and_Integration_test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
     types: [closed]
     branches:
-      - develop
+      - Build_Deploy
```
Comment (yashmeet29) [inline .github/workflows/deploy_and_Integration_test.yml]: Change this back to develop

File: .github/workflows/deploy_and_Integration_test.yml [hunk 2]
```diff
@@ -25,12 +25,15 @@ jobs:
           
       - name: Login to Cloud Foundry
         run: |
-          wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
-          echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
-          sudo apt-get update
-          sudo apt-get install cf8-cli
+          sudo apt-key del 172B5989FCD21EF8
```
Comment (yashmeet29) [inline .github/workflows/deploy_and_Integration_test.yml]: Is this a one-time command or we need to run it everytime?

Comment (vibhutikumar07) [reply .github/workflows/deploy_and_Integration_test.yml]: Not required overtime so removed the part

File: .github/workflows/deploy_and_Integration_test.yml (hunk 1 above)
Comment (vibhutikumar07) [reply .github/workflows/deploy_and_Integration_test.yml]: done

**Review by yashmeet29 — APPROVED** [review]

**Pull request merged**